tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -85,6 +85,25 @@ def load_conditions(session):
 
     return conditions_list
 
+def read_asic_name(hwsku):
+    asic_name_file = os.path.dirname(__file__) + ASIC_NAME_PATH
```

Comment:
Is the `ASIC_NAME_PATH` defined somewhere?

Reply:
oh, sorry, I forget to paste this.